docs/draft/rest_api.md

```diff
@@ -0,0 +1,111 @@
+# First Draft for the REST API
+All calls and requests return data as JSON.
+## Data that needs to be provided by the API
+- current status of the transformer
+  ```
+  GET status/
+  ```
+  **returns:**
+  ```json
+  {
+    "state": "error|transformation|idle",
+    "current_transformations": [1,2,3]
+  }
+  ```
+  *state*: current status of the transformer
+  *current_transformations:* ids of the active or queued transformations
+- the result of the transformation: file/archive
+  ```
+  GET transformation/{id}
+  ```
+  **on success returns:**
```

Comment:
This has to be redone. If i understood this correct  this should return the result of the transformation once its done. Something like this cannot be done in a string. Lets say the resulting archiveis 100 MB large. then we should not put the data into a json file. Maybe file points to some HTTP path with wich the user can download the archive.